systemtest/src/test/java/io/strimzi/systemtest/kafka/listeners/ListenersST.java

```diff
@@ -131,18 +130,18 @@ public class ListenersST extends AbstractST {
         }
     )
     void testSendMessagesPlainAnonymous() {
-        final TestStorage testStorage = new TestStorage(KubeResourceManager.get().getTestContext());
+        final TestStorage testStorage = new TestStorage(resourceManager.getTestContext());
```

Comment:
We should be consistent in RM usage - either use `resourceManager` everywhere (from AbstractST) or use `KubeResourceManager.get()` everywhere.

Reply:
I think I will remove the `resourceManager` and keep just `KubeResourceManager.get()` everywhere - so it will be consistent in ST classes, but also code classes.

Reply:
Changed it to `KubeResourceManager.get()` everywhere and removed `resourceManager` from `AbstractST`